commitizen/config/__init__.py

```diff
@@ -3,17 +3,39 @@
 from pathlib import Path
 
 from commitizen import defaults, git
+from commitizen.exceptions import ConfigFileNotFound
 
 from .base_config import BaseConfig
 from .json_config import JsonConfig
 from .toml_config import TomlConfig
 from .yaml_config import YAMLConfig
 
 
-def read_cfg() -> BaseConfig:
+def read_cfg(filepath: str | None = None) -> BaseConfig:
     conf = BaseConfig()
 
     git_project_root = git.find_git_project_root()
+
+    if filepath is not None:
+        given_cfg_path = Path(filepath)
```

Comment:
```suggestion
        cfg_path = Path(filepath)
```

cfg_path alone should be expressive enough